test/com/google/javascript/jscomp/ProcessCommonJSModulesTest.java

```diff
@@ -251,38 +251,4 @@ public void testEs6ObjectShorthand() {
         "goog.provide('module$test');"
             + "module$test = { prop: 'value', foo() { console.log('bar'); }}" );
   }
-
-  public void testSortInputs() throws Exception {
-    SourceFile a = SourceFile.fromCode("a.js", "require('b');require('c')");
```

Comment:
Is this test really obsolete? Do we have other code testing the pre-order traversal sort here?